node/Cargo.toml

```diff
@@ -20,7 +20,9 @@ futures = { workspace = true }
 serde_json = { workspace = true }
 
 # Local
-pop-runtime = { workspace = true }
+pop-runtime-devnet = { workspace = true }
```

Comment:
Be nice to use `.workspace = true style`

Reply:
Can refactor in a different PR

Reply:
Was quick to do so addressed in a PR to this one so its done.